evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -288,18 +297,18 @@ contract Operator is
   }
 
   /**
-   * @notice Retrive a list of forwarders
-   * @return array of addresses
+   * @notice Retrive the forwarder
+   * @return address
    */
-  function getForwarders()
+  function getForwarder()
     external
     view
     override
     returns (
-      address[] memory
+      address
```

Comment:
Maybe we should just make the return type OperatorForwarder instead of address? Makes it easier for interactions in solidity, and gets translated into an address anyway for anything querying off-chain.

Reply:
I'm putting a pin in this for now since it would require altering existing and adding more interfaces, but maybe this is something we should be adding to the Solidity style guide:

"Where possible, return contract type variables instead of address type variables"